2-mandatory/3-bush-berries.js

```diff
@@ -22,6 +22,15 @@
 */
 
 function isBushSafe(berryArray) {
```

Comment:
excellent! great use of anonymous arrow functions to make coding concise but still easy to follow.